aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md

```diff
@@ -37,9 +37,9 @@ The following diagram from [a Microsoft Patterns and Practices e-book about cont
 
 Automating these processes in Azure is easy. Because you're running everything in the cloud, you don't have to buy or manage servers for your builds or your test environments. And you don't have to wait for a server to be available to do your testing on. With every build that you do, you could spin up a test environment in Azure using your automation script, run acceptance tests or more in-depth tests against it, and then when you're done just tear it down. And if you only run that server for 2 hours or 8 hours or a day, the amount of money that you have to pay for it is minimal, because you're only paying for the time that a machine is actually running. For example, the environment required for the Fix it application basically costs about 1 cent per hour if you go one tier up from the free level. Over the course of a month, if you only ran the environment an hour at a time, your testing environment would probably cost less than a latte that you buy at Starbucks.
 
-## Visual Studio Online
+## Visual Studio Team Services (VSTS)
 
-One of the things we're working on with Visual Studio Online (VSO), formerly known as Team Foundation Service, is to make it work really well with continuous integration and delivery. Here are some key features of VSO:
+VSTS provided all the features and aspects of application development from planning to deployment. Here are some key features of VSTS:
```

Comment:
Thanks @GuardRex I do agree and I should have been more careful about "present tense". "VSTS provides a number of features to assist you with application development from planning to deployment."